src/views/List.jsx

```diff
@@ -23,18 +25,38 @@ export function List({ data }) {
 		setSearchTerm('');
 	};
 
+	const renderedListLength = renderedList.length;
+
 	return (
 		<>
-			<form onSubmit={clearSearchField}>
-				<label htmlFor="search-filter">Item name:</label>
-				<input
-					type="text"
-					id="search-filter"
-					value={searchTerm}
-					onChange={(e) => setSearchTerm(e.target.value)}
-				/>
-				<button type="submit"> clear </button>
-			</form>
+			{renderedListLength === 0 && (
+				<>
+					<h2>Your list currently has no items. </h2>
+					<h3>Click on the add first item button to start your list.</h3>
+					<Link to="/add-item">
+						<button> add first item</button>
+					</Link>
+				</>
+			)}
+			{renderedListLength > 0 && (
+				<>
```

Comment:
Good job. You see this all the time in React, with the short-circuiting. You have choices here. You could also choose to render with an if/else (who would, but hey, it's an option), OR a ternary. `{renderedListLength > 0? (return some JSX) : (return this other JSX)}`

Reply:
I started writing this exact thing before I saw that Dana had already said it! Great minds. 

IMO, all of these choices for how to do conditional rendering come down to readability. Do the thing that makes the code easiest to understand!

Reply:
Thanks, @danathedev and @mxmason, for the guidance here! This has been refactored with a ternary.